src/infer/inferOs.js

```diff
@@ -11,7 +11,7 @@ function inferPlatform() {
 
 function inferArch() {
   const arch = os.arch();
-  if (arch !== 'ia32' && arch !== 'x64') {
+  if (arch !== 'ia32' && arch !== 'x64' && arch !== 'arm') {
```

Comment:
@mdh34 so, the command-line flag is `armv7l`, but here `os.arch()` returns just `arm`? (Just checking, if yes it's alright).

Reply:
@ronjouch Im the maintainer of lysmarineOS and I looking to use it to display the local run of the signalK server. and other web based local services as freeboard-sk pypilot, maybe some sensors/stat display.